charts/app-config/values-integration.yaml

```diff
@@ -1190,6 +1190,50 @@ govukApplications:
               name: signon-token-government-frontend-publishing-api
               key: bearer_token
 
+  - name: govuk-chat
+    helmValues:
+      dbMigrationEnabled: true
+      workerEnabled: true
+      ingress:
+        enabled: true
+        annotations:
+          <<: [*alb-ingress-group-backend, *alb-ingress-defaults]
```

Comment:
Not sure I have used the correct group here, or if this one is needed for our case `alb-ingress-waf-ruleset-www` instead.

Reply:
So here we Basically have the choice between the cache_public_web_acl and the backend_public_web_acl, differents access and rate limiting rules in AWS WAF, we might want to create our own WAF ACL for govuk.chat as it might make sense to have a separate ACL to segregate the rate limiting and logging that we might want to do.
Something to discuss with platform-engineering I'd say

Reply:
Thanks for arranging the call with platform-engineering Roch, at least we now know that for integration this config is ok, no need to create other acls